### PR TITLE
[KOGITO-4858] Automatic build of examples

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-4858

Dependent PR: https://github.com/kiegroup/kogito-operator/pull/832

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1525,6 +1526,8 @@ github.com/prometheus/tsdb v0.8.0/go.mod h1:fSI0j+IUQrDd7+ZtR9WKIGtoYAYAJUKcKhYL
 github.com/quasilyte/go-consistent v0.0.0-20190521200055-c6f3937de18c/go.mod h1:5STLWrekHfjyYwxBRVRXNOSewLJ3PWfDJd1VyTS21fI=
 github.com/quasilyte/go-ruleguard v0.1.2-0.20200318202121-b00d7a75d3d8/go.mod h1:CGFX09Ci3pq9QZdj86B+VGIdNj4VyCo2iPOGS9esB/k=
 github.com/quobyte/api v0.1.2/go.mod h1:jL7lIHrmqQ7yh05OJ+eEEdHr0u/kmT1Ff9iHd+4H6VI=
+github.com/radtriste/kogito-operator v0.0.0-20210406191637-0c6a48be5404 h1:d3xCC5tXl36eqlAeXhSYcn0Y7z1QogPdpj+bO8OummI=
+github.com/radtriste/kogito-operator v0.0.0-20210406191637-0c6a48be5404/go.mod h1:s7IgEpGfN6Blwbsm1nEkGcnQxmSeMWJJeeilfU8cDW8=
```
Comment (sutaakar) [inline go.sum:1530]: is it a leftover?

**Review by sutaakar — APPROVED** [review]
Looks good to me

File: go.sum (hunk 1 above)
Comment (radtriste) [reply go.sum:1530]: seems so ... nice catch, gonna remove that

**Comment by radtriste** [comment]
@vaibhavjainwiz @sutaakar @ricardozanini I am stuck on that one
```
Error while creating Prometheus CR: no kind is registered for the type v1.Prometheus
```
but we do register it in the `meta.go` => https://github.com/kiegroup/rhpam-kogito-operator/blob/master/meta/meta.go#L64

Am I missing something ?

**Comment by sutaakar** [comment]
I think it is going to be addressed in https://github.com/kiegroup/rhpam-kogito-operator/pull/15
So ok to merge from my side.

**Comment by radtriste** [comment]
> I think it is going to be addressed in #15
> So ok to merge from my side.

**Pull request merged**